app/routes/contact/components/ContactForm.tsx

```diff
@@ -58,9 +58,8 @@ const ContactForm = (props: Props) => {
       <p>
         Dette skjemaet kan benyttes for å kontakte Abakus sine komiteer eller
         Hovedstyret. Dersom du har opplevd avvik eller ubehagelige hendelser i
-        forbindelse med Abakus, ønsker vi gjerne at dette meldes fra om her.
-        Send oss gjerne også spørsmål, feedback eller ting du ønsker å fortelle
-        oss, for eksempel om du ønsker å avtale tid for innsyn i HSP-avtalen.
+        forbindelse med Abakus, ønsker vi at dette meldes fra om! Send oss
+        gjerne også spørsmål, tilbakemelding eller ting du ønsker å fortelle.
```

Comment:
This is a bit unclear. This form should not be used for "ubehagelige hendelser and avvik". It should be clear that this should go through the avvik.abakus.no.

Reply:
In general. I think removing anything about avvik in this form, and adding a clear, separate section at the top that says that all varsling should be send to the portal would be clearer and easier to understand.

Reply:
I think the bit about feedback should be kept. If someone wants to give anonymous feedback that does not need a formal notice/warning, such as that the student areas are intimidating for new students, that should go through the contact form.

Reply:
> This form should not be used for "ubehagelige hendelser and avvik".

That's why I removed "her". This sentence simply says that we both want and encourage people to contact - but not how, since that's below. There's a lot of text on this page already, and I don't want to duplicate anything, while also being very clear.

Reply:
I believe I made it clearer now by moving the section about avvik to the top. I also kept the part about feedback.

I think the text is clear and concise now. It's not that pretty with a lot of text, but in this case functionality is much better than looks.